### PR TITLE
Validated cat.health Req & Res

### PR DESCRIPTION
**Pull request by philkra**
as titled.

### CODE REVIEW

File: specification/specs/cat/cat_health/CatHealthRecord.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ class CatHealthRecord {
   relo: string
   shards: string
   status: string
-  timestamp: string
+  timestamp?: string
```
Comment (delvedor) [inline specification/specs/cat/cat_health/CatHealthRecord.ts:31]:
```suggestion
  timestamp?: DateString
```

File: specification/specs/cat/cat_health/CatHealthRecord.ts [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 class CatHealthRecord {
   cluster: string
-  epoch: string
+  epoch?: string
```
Comment (delvedor) [inline specification/specs/cat/cat_health/CatHealthRecord.ts:22]:
```suggestion
  epoch?: EpochMillis
```

**Comment by delvedor** [comment]
https://github.com/elastic/elastic-client-generator/pull/126#pullrequestreview-593917246 still needs to be addressed. Can you regenerate the output as well?

**Comment by philkra** [comment]
@delvedor : https://github.com/elastic/elastic-client-generator/pull/126#issuecomment-782008460 --> done

**Review by delvedor — APPROVED** [review]
LGTM

**Pull request merged**